documentation/modules/upgrading/con-downgrade-target-version.adoc

```diff
@@ -2,17 +2,42 @@
 //
 // assembly-downgrading-kafka-versions.adoc
 
-[id='com-target-downgrade-version-{context}']
+[id='con-target-downgrade-version-{context}']
 
-= Target downgrade version
+= Kafka version compatibility for downgrades
 
-How the Cluster Operator handles a downgrade operation depends on the `log.message.format.version`.
+Kafka downgrades are dependent on compatible current and target xref:ref-kafka-versions-{context}[Kafka versions],
+and the state at which messages have been logged.
 
-* If the target downgrade version of Kafka has the same `log.message.format.version` as the current version, the Cluster Operator downgrades by performing a single rolling restart of the brokers.
-* If the target downgrade version of Kafka has a different `log.message.format.version`, downgrading is only possible if the running cluster has _always_ had `log.message.format.version` set to the version used by the downgraded version.
-+
+You cannot revert to the previous Kafka version if the `inter.broker.protocol.version` differs, or messages have been added
```

Comment:
This sentence on its own (I know is clarified in a couple of sentences time) might make the reader think they can just change the IBP first. 

```suggestion
You cannot revert to the previous Kafka version if that version don't support any of the `inter.broker.protocol.version` settings which have ever been used in that cluster, or messages have been added
```